app/jobs/get_usps_proofing_results_job.rb

```diff
@@ -623,6 +623,7 @@ def log_enrollment_updated_analytics(enrollment:, enrollment_passed:, enrollment
       reason: reason,
```

Comment:
I looked at every path- the only case where I don't see an analytic logged for handle_unexpected_response when it is cancelled. It looks like it is missing the entire analytic event. 

I looked at: handle_success, handle_unsupported_id_type, handle_expired_status_update, handle_failed_status, handle_unsupported_secondary_id.

Reply:
This commit is intended to allow us to track how long it takes verified uses to complete proofing. It is my understanding that `handle_unexpected_response` does not result in a verified profile or change the state of the profile at all. I'll leave that fix to a different change request.

Reply:
Ah, yes. It only changes the enrollment- not the profile. Resolving.

Reply:
I opened #11195 to address the logging issue